app/twit.hoon

```diff
@@ -195,11 +195,11 @@
 ::
 ++  compat  |=({usr/(unit iden) req/(unit iden)} ?~(req & =(usr req)))
 ::
-::  .^(twit-feed %gx /=twit=/~/home/urbit_test)
-::  .^(twit-stat %gx /=twit=/~./post/0vv0old.0post.hash0.0000)
-++  peek
-  |=  {ren/care pax/path}  ^-  (unit (unit gilt))
-  ?>  ?=($x ren)  ::  others unsupported
+::   /+  twitter
+::  .^((list post:twitter) %gx /=twit=/home/urbit_test/twit-feed)
+::  .^(post:twitter %gx /=twit=/post/0vv0old.0post.hash0.0000/twit-feed)
```

Comment:
I can't the `$post` portion of the namespace to work, and I haven't figured out what's going on yet. But it'll need a different mark too